tests/suites/test_suite_pk.function

```diff
@@ -15,7 +15,7 @@
 #include <limits.h>
 #include <stdint.h>
 
-/* Needed only for test case data under #if defined(MBEDTLS_USE_PSA_CRYPTO),
+/* Needed only for test case data using PSA API macros/types because
```

Comment:
Minor: "because but" (over the line break). Actually I'd remove the comment entirely, I think these days we don't need to justify including `psa/crypto.h`.